### PR TITLE
demo to saumya

### PR DESCRIPTION
**Pull request by skairamk**

### CODE REVIEW

**Comment by saumya-pivotaldesign** [comment]
looks good

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# hztb-ios
+# hztb-ios - demo to saumya
```
Comment (skairamk) [inline README.md:1]: why is this needed?

Comment (skairamk) [inline README.md:1]: just for demo
